README.md

```diff
@@ -4,7 +4,7 @@ Netuitive Cookbook (Chef)
 [![Build Status](https://travis-ci.org/Netuitive/chef-netuitive.svg?branch=master)](https://travis-ci.org/Netuitive/chef-netuitive) [![Join the chat at https://gitter.im/Netuitive/chef-netuitive](https://badges.gitter.im/Netuitive/chef-netuitive.svg)](https://gitter.im/Netuitive/chef-netuitive?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/Netuitive/chef-netuitive/master/LICENSE)
 
 A cookbook to automate the installation and configuration of the Netuitive Linux agent. For more
-information on the Netuitive Linux Agent, see the [help docs](https://help.netuitive.com/Content/Misc/Datasources/Netuitive/new_netuitive_datasource.htm) or contact Netuitive support at [support@netuitive.com](mailto:support@netuitive.com).
+information on the Netuitive Linux Agent, see the [help docs](https://docs.virtana.com/en/linux-agent.html) or contact Netuitive support at [support@netuitive.com](mailto:support@netuitive.com).
```

Comment:
Please check for `support@netuitive.com` links as well (anything with `netuitive.com` and `metricly.com`). Reach out to support to get the new address.